### PR TITLE
disable auto-prompt mode for cloudant. also update README.md file

### PR DESCRIPTION
**Pull request by lehoanganh**
FYI: @domdom82 @rabbah

### CODE REVIEW

File: ansible/README.md [hunk 1]
```diff
@@ -87,12 +87,18 @@ ansible-playbook -i environments/<environment> couchdb.yml -e mode=initdb
 ansible-playbook -i environments/<environment> openwhisk.yml
 ```
 
-You need to run "initdb" on couch **every time** you deploy couchdb to initialize the db.
+You need to run `initdb` on couchdb **every time** you deploy couchdb to initialize the db.
+
+**Hint:** The `initdb` mode will ask for your confirmation to drop the database. To move on you need to enter `CTRL-C + C`. To abort you need to enter `CTRL-C + A`. You can skip this confirmation step by adding `-e prompt_user=false` to the command (only for `couchdb`, not for `cloudant`)
+
```
Comment (rabbah) [inline ansible/README.md]: The doc still mentions prompt bypassing above and below.

cloudant should be Cloudant

couchdb should be CouchDB

Comment (lehoanganh) [inline ansible/README.md]: with couchdb, one can bypass the prompt (needed for travis). with cloudant, one can not do it anymore, you know why ;)

Comment (rabbah) [inline ansible/README.md]: Why? Do you distinguish between a local couch and a persistent one?

Comment (markusthoemmes) [inline ansible/README.md]: Prompting should be standard configuration for every database there is. They can all have a "don't prompt me, i know what i'm doing" switch though.

**Comment by rabbah** [comment]
> they can all have a "don't prompt me, i know what i'm doing" switch though.

negative.

**Comment by lehoanganh** [comment]
copy and paste from @domdom82 

we just discussed a possible solution for the issue:
1. remove initdb mode
2. move create immortal db stuff to common initdb.yml that gets included by both cloudant and couch roles
3. initdb.yml does this:
```
1. check if subjects db exists
2. if yes, do nothing
3. if no, create it. add initial keys
```

This way there is no way ever that a user can accidentally drop subjects db by means of our playbooks.
If the user really wants to drop the subjects db, they have to manually log into cloudant and manually drop it there. No automatic way exists anymore.

Another pull will be on the go...

**Pull request merged**